quarkchain/cluster/cluster_config.py

```diff
@@ -171,7 +203,7 @@ def __init__(self):
         slave_config = SlaveConfig()
         slave_config.PORT = 38000
         slave_config.ID = "S0"
-        slave_config.CHAIN_MASK_LIST = [ChainMask(1)]
+        slave_config.FULL_SHARD_ID_LIST = [2]
```

Comment:
Should FULL_SHARD_ID_LIST be [1] instead of [2] here?

Reply:
updated. though I don't see this default config being used any where